docs/1.concepts/abstraction/chain-signatures.md

```diff
@@ -29,38 +30,31 @@ _Diagram of a chain signature in NEAR_
 
 ### 1. Create Payload
 
-The first step is to construct a payload (transaction, message, data, etc.) for the target blockchain platform.
+The first step is to construct a payload (transaction, message, data, etc.) for the target blockchain platform. This variates depending on the target blockchain, but in general, it's a hash of the message or transaction to be signed.
 
-This can be performed in the following steps:
-<!-- TODO -->
-
-1.
-
-2.
-
-3.
+<hr class="subsection" />
 
 ### 2. Signature Request
 
-Once a payload is created and ready to sign, a signature request is made by calling `sign` the deployed smart contract `multichain.near`. This method takes two parameters:
-  - **payload:** A payload (transaction, message, data, etc.) signed by a NEAR account
```

Comment:
The sender must first sign the payload before the next step.

Reply:
for what I saw in the code the NEAR account does not need to sign the payload, it is sent raw, and the MPC network signs it

Reply:
Hmmm, I could have sworn I heard "The payload needs to be first signed by the NEAR account and then again by MPC" and it stuck in my head as an interesting notion of twice signed transaction. I probably misunderstood. 

@DavidM-D @volovyks can you confirm that I'm mistaken here?